### PR TITLE
add google-cloud-secret-manager==2.12.6 to packages.ini

### PR DESCRIPTION
**Pull request by mwarkentin**

### CODE REVIEW

File: packages.ini [hunk 1]
```diff
@@ -301,9 +308,12 @@ python_versions = <3.11
 python_versions = <3.11
 [grpcio==1.48.1]
 python_versions = <3.11
+[grpcio==1.51.1]
+python_versions = <3.11
```
Comment (asottile-sentry) [inline packages.ini:312]: can remove this line -- grpcio has wheels for 3.11 now

**Review by bmckerry — APPROVED** [review]

File: packages.ini (hunk 1 above)
Comment (mwarkentin) [reply packages.ini:312]: @asottile-sentry this was added automatically by `add_pkg` - do we need an update there?

Comment (asottile-sentry) [reply packages.ini:312]: `add_pkg` copies over the settings from previous versions as a convenience -- newer versions _sometimes_ have different requirements than previous versions

Comment (mwarkentin) [reply packages.ini:312]: 👍🏻 was looking through the code and figured the same - should I just remove from this latest version or do the older versions have 3.11 wheels as well now?

Comment (asottile-sentry) [reply packages.ini]: just the latest version :+1:

**Pull request merged**